R/plot_sankey.R

```diff
@@ -2,6 +2,7 @@
 #'
 #' @param data data.frame. Should have the same format as output of
 #'   `prep_sankey()`
+#' @param by_group Character. Vector of length 1. Variable to group by.
```

Comment:
(very) NIT: but the name `by_group` seems like it should take a boolean?

Reply:
any other ideas for naming this? `group` does not seem much better.. `.by` would seem odd here?

Reply:
`sankey_group`?

Reply:
it is being used for all the different plots... so it would have to be `<plot>_group` then.. not sure

Reply:
naming things has go to be the worst exercise haha. I think `group_var` is good. Weirdly enough that is what I wanted to use in `pacta.multi.loanbook.analysis` before opting for `.by`. But since it does something slightly different there, I would be find with using `group_var` here

Reply:
There are only two hard things in Computer Science: cache invalidation and naming things.

-- Phil Karlton